Feature/site/wal 4

Waleed9042 opened the pull request:
I have tried my best to cover all the components the one left are on intention and will be done when working on that particular feature.
I hope you will like the work.
Thanks

x365Project commented:
did you see my comments?

On Wed, Mar 31, 2021 at 9:17 AM Waleed9042 ***@***.***> wrote:

> I have tried my best to cover all the components the one left are on
> intention and will be done when working on that particular feature.
> I hope you will like the work.
> Thanks
> ------------------------------
> You can view, comment on, or merge this pull request online at:
>
>   https://github.com/x365Project/appglue/pull/128
> Commit Summary
>
>    - [wal-4]: NavFrame (appRoot) story and tests
>    - [wal-4]: added test and story for topBar (vertical layout)
>    - [wal-4]: added tests and story for topBarNav (horizontal layout)
>    - [wal-4]: added story and basic tests for sideBarNav (vertical layout)
>    - [wal-4]: add tests and stories for common header component
>    - [wal-4]: add story and tests for chat options
>    - [wal-4]: Add notification component tests and story
>    - [wal-4]: Add unit tests and story for profileComponent
>    - [wal-4]: Add tests and stories for remaing components
>
> File Changes
>
>    - *M* appglue.site/src/Components/NavFrame/NavFrame.stories.tsx
>    <https://github.com/x365Project/appglue/pull/128/files#diff-bb1263fca03c428870337dd83a9fe6c19175048e4979a2de3b3553d8465ed8a2>
>    (20)
>    - *M* appglue.site/src/Components/NavFrame/NavFrame.test.tsx
>    <https://github.com/x365Project/appglue/pull/128/files#diff-d156d4a563f2f59665dd2017cc13391c3ca8ee4b77858ce3558acb7b4eaa5bdb>
>    (268)
>    - *M*
>    appglue.site/src/Components/NavFrame/__snapshots__/NavFrame.test.tsx.snap
>    <https://github.com/x365Project/appglue/pull/128/files#diff-08fa82a34b919fca2ff0173cf0a93819fd09ba451928f29f6b6944dac994fc3f>
>    (7780)
>    - *A*
>    appglue.site/src/Components/TopBar/SideNavBar/SideBarNav.stories.tsx
>    <https://github.com/x365Project/appglue/pull/128/files#diff-7d596f4defe5e78d4b92100080e56aa9c7d3b60bd987e6eb93c5e27e6208de2a>
>    (40)
>    - *M* appglue.site/src/Components/TopBar/SideNavBar/SideNavBar.test.tsx
>    <https://github.com/x365Project/appglue/pull/128/files#diff-12031edfefca2093aaf5df1f118ede071ff46273c61355c7b98243a8f4c0ae98>
>    (239)
>    - *M*
>    appglue.site/src/Components/TopBar/SideNavBar/__snapshots__/SideNavBar.test.tsx.snap
>    <https://github.com/x365Project/appglue/pull/128/files#diff-cf4e434c7cecc14b4e66586ffd3b6105b15f52f0e1adeba33db479a73e8727c4>
>    (666)
>    - *A* appglue.site/src/Components/TopBar/TopBar.stories.tsx
>    <https://github.com/x365Project/appglue/pull/128/files#diff-70a6889eed71ee729bcf36bbbc788c1e7812f785bc550b9d2cb3a1db05a2eb25>
>    (32)
>    - *M* appglue.site/src/Components/TopBar/TopBar.test.tsx
>    <https://github.com/x365Project/appglue/pull/128/files#diff-c71bc0c6afdbce49733febeb9e6112301483597c6b3ece4c58dba1b74b062c15>
>    (237)
>    - *M*
>    appglue.site/src/Components/TopBar/__snapshots__/TopBar.test.tsx.snap
>    <https://github.com/x365Project/appglue/pull/128/files#diff-ca584b16dd1c96a5b9455ac33eec057c560195e2daf18850d68c60f4df9ab9fd>
>    (3904)
>    - *A* appglue.site/src/Components/TopBarNav/TopBarNav.stories.tsx
>    <https://github.com/x365Project/appglue/pull/128/files#diff-57395f420b75e2f1952218959da13f509280c2d589ff44cdeb169f079cfbe771>
>    (33)
>    - *M* appglue.site/src/Components/TopBarNav/TopBarNav.test.tsx
>    <https://github.com/x365Project/appglue/pull/128/files#diff-c4349db6035394e0e8f024716e4b96c498d73274a76c29f00788f137b94da327>
>    (242)
>    - *M*
>    appglue.site/src/Components/TopBarNav/__snapshots__/TopBarNav.test.tsx.snap
>    <https://github.com/x365Project/appglue/pull/128/files#diff-d4616297803ab8ef01256c0246f2745fa6892e3b6490469b08d1ce4d50317894>
>    (4828)
>    - *A*
>    appglue.site/src/Components/common/AppHeader/AppHeader.stories.tsx
>    <https://github.com/x365Project/appglue/pull/128/files#diff-81183aac2e240bb2db21182ad06445bfbdcd83220108f2c5eb7dcba1e425fafd>
>    (45)
>    - *M* appglue.site/src/Components/common/AppHeader/AppHeader.test.tsx
>    <https://github.com/x365Project/appglue/pull/128/files#diff-171672af6c4c79471669a6f4ac4372008daa7936f17f33b82e6af1a79f12d51d>
>    (223)
>    - *M*
>    appglue.site/src/Components/common/AppHeader/__snapshots__/AppHeader.test.tsx.snap
>    <https://github.com/x365Project/appglue/pull/128/files#diff-dd6007b0c64c586ff144908adfc4f3cf017772ef7411929d1f01d8c20916bdb6>
>    (4756)
>    - *A*
>    appglue.site/src/Components/common/ChatOptions/ChatOptions.stories.tsx
>    <https://github.com/x365Project/appglue/pull/128/files#diff-8f191b868b60a5d065fbf734ecdd0ebd6cfa85ed568d190f75203e897d526fdb>
>    (17)
>    - *A*
>    appglue.site/src/Components/common/ChatOptions/ChatOptions.test.tsx
>    <https://github.com/x365Project/appglue/pull/128/files#diff-0e2ef8e3d1be4bd994dfef22b4cf80475a1a3ac406ae3188dd99eb7327d18b39>
>    (24)
>    - *A*
>    appglue.site/src/Components/common/ChatOptions/__snapshots__/ChatOptions.test.tsx.snap
>    <https://github.com/x365Project/appglue/pull/128/files#diff-b8a56be555b7009aa244907412b9c03a61df92a616fe11273d457731acca80a7>
>    (229)
>    - *A*
>    appglue.site/src/Components/common/MainContentArea/ContentArea.test.tsx
>    <https://github.com/x365Project/appglue/pull/128/files#diff-03ceb229876408b0c264e535002fbc1f05f5a69bc655f974cf83f813dfcb2e22>
>    (12)
>    - *M*
>    appglue.site/src/Components/common/MainContentArea/ContentArea.tsx
>    <https://github.com/x365Project/appglue/pull/128/files#diff-6e17b681694b0253f30461b0a30968c880ffb0272a0218e57df9e9203e4c20fd>
>    (2)
>    - *A*
>    appglue.site/src/Components/common/MainContentArea/MainContentArea.stories.tsx
>    <https://github.com/x365Project/appglue/pull/128/files#diff-58552dd7d4fb4aed2359d51d18c79daabd798360ad895647bf73e454fc3639ae>
>    (23)
>    - *A*
>    appglue.site/src/Components/common/MainContentArea/__snapshots__/ContentArea.test.tsx.snap
>    <https://github.com/x365Project/appglue/pull/128/files#diff-f482df38435eb6a44ed8cefae9ca6bd355d2493a0523edf018b789003eae3d05>
>    (15)
>    - *A*
>    appglue.site/src/Components/common/NotificationsOptions/NotificationsOptions.stories.tsx
>    <https://github.com/x365Project/appglue/pull/128/files#diff-aed5357c5a961b21bd674849492163685ef78e71289a2ec11baa99056da7e402>
>    (16)
>    - *A*
>    appglue.site/src/Components/common/NotificationsOptions/NotificationsOptions.test.tsx
>    <https://github.com/x365Project/appglue/pull/128/files#diff-fe6fda07a58fe75d5b2a1a7632eba0e1054eb536717eded84a6808624822d1ac>
>    (34)
>    - *A*
>    appglue.site/src/Components/common/NotificationsOptions/__snapshots__/NotificationsOptions.test.tsx.snap
>    <https://github.com/x365Project/appglue/pull/128/files#diff-2c2fd4f00900e67fe985cfbc9f81aeb2f2f12423d38f0ca011246a3ab03ab2c4>
>    (304)
>    - *A*
>    appglue.site/src/Components/common/Profile/ProfileComponent.stories.tsx
>    <https://github.com/x365Project/appglue/pull/128/files#diff-63f5514754ee98c4501fe63fcd676157fa2f1dd568ee020dc4d7636c0efe2964>
>    (16)
>    - *A*
>    appglue.site/src/Components/common/Profile/ProfileComponent.test.tsx
>    <https://github.com/x365Project/appglue/pull/128/files#diff-b2020e2abe0aa2e3b7f0e7875169512c8a074b99eff38acc34a85901ff12037e>
>    (22)
>    - *A*
>    appglue.site/src/Components/common/Profile/__snapshots__/ProfileComponent.test.tsx.snap
>    <https://github.com/x365Project/appglue/pull/128/files#diff-9271f9a22569ed11e0bdea2647260b7222c9b845f0af7a27f35c7222d15212b4>
>    (131)
>    - *A*
>    appglue.site/src/Components/common/SearchBar/SearchBar.stories.tsx
>    <https://github.com/x365Project/appglue/pull/128/files#diff-58398dbcc554dc52a843d181031411c3867e0302982a990b7f9159e6bb41cbec>
>    (13)
>    - *A*
>    appglue.site/src/Components/common/SettingsPanel/SettingsPanel.stories.tsx
>    <https://github.com/x365Project/appglue/pull/128/files#diff-777bcd72bbcefae203b8fa67889ee8417e5fbde06c53640de77acb49dcc3c2ca>
>    (25)
>    - *A*
>    appglue.site/src/Components/common/SingleSetting/SingleSetting.stories.tsx
>    <https://github.com/x365Project/appglue/pull/128/files#diff-1656010f57225e2e86ea3ebf9b33849685d01d9ea7a466f5b992665fe9da8472>
>    (19)
>
> Patch Links:
>
>    - https://github.com/x365Project/appglue/pull/128.patch
>    - https://github.com/x365Project/appglue/pull/128.diff
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/x365Project/appglue/pull/128>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARCCXOTXWAWUMOMBTHWRX5DTGMOINANCNFSM42EUHCWQ>
> .
>

Waleed9042 commented:
> did you see my comments?
> […](#)
> On Wed, Mar 31, 2021 at 9:17 AM Waleed9042 ***@***.***> wrote: I have tried my best to cover all the components the one left are on intention and will be done when working on that particular feature. I hope you will like the work. Thanks ------------------------------ You can view, comment on, or merge this pull request online at: #128 Commit Summary - [wal-4]: NavFrame (appRoot) story and tests - [wal-4]: added test and story for topBar (vertical layout) - [wal-4]: added tests and story for topBarNav (horizontal layout) - [wal-4]: added story and basic tests for sideBarNav (vertical layout) - [wal-4]: add tests and stories for common header component - [wal-4]: add story and tests for chat options - [wal-4]: Add notification component tests and story - [wal-4]: Add unit tests and story for profileComponent - [wal-4]: Add tests and stories for remaing components File Changes - *M* appglue.site/src/Components/NavFrame/NavFrame.stories.tsx <https://github.com/x365Project/appglue/pull/128/files#diff-bb1263fca03c428870337dd83a9fe6c19175048e4979a2de3b3553d8465ed8a2> (20) - *M* appglue.site/src/Components/NavFrame/NavFrame.test.tsx <https://github.com/x365Project/appglue/pull/128/files#diff-d156d4a563f2f59665dd2017cc13391c3ca8ee4b77858ce3558acb7b4eaa5bdb> (268) - *M* appglue.site/src/Components/NavFrame/__snapshots__/NavFrame.test.tsx.snap <https://github.com/x365Project/appglue/pull/128/files#diff-08fa82a34b919fca2ff0173cf0a93819fd09ba451928f29f6b6944dac994fc3f> (7780) - *A* appglue.site/src/Components/TopBar/SideNavBar/SideBarNav.stories.tsx <https://github.com/x365Project/appglue/pull/128/files#diff-7d596f4defe5e78d4b92100080e56aa9c7d3b60bd987e6eb93c5e27e6208de2a> (40) - *M* appglue.site/src/Components/TopBar/SideNavBar/SideNavBar.test.tsx <https://github.com/x365Project/appglue/pull/128/files#diff-12031edfefca2093aaf5df1f118ede071ff46273c61355c7b98243a8f4c0ae98> (239) - *M* appglue.site/src/Components/TopBar/SideNavBar/__snapshots__/SideNavBar.test.tsx.snap <https://github.com/x365Project/appglue/pull/128/files#diff-cf4e434c7cecc14b4e66586ffd3b6105b15f52f0e1adeba33db479a73e8727c4> (666) - *A* appglue.site/src/Components/TopBar/TopBar.stories.tsx <https://github.com/x365Project/appglue/pull/128/files#diff-70a6889eed71ee729bcf36bbbc788c1e7812f785bc550b9d2cb3a1db05a2eb25> (32) - *M* appglue.site/src/Components/TopBar/TopBar.test.tsx <https://github.com/x365Project/appglue/pull/128/files#diff-c71bc0c6afdbce49733febeb9e6112301483597c6b3ece4c58dba1b74b062c15> (237) - *M* appglue.site/src/Components/TopBar/__snapshots__/TopBar.test.tsx.snap <https://github.com/x365Project/appglue/pull/128/files#diff-ca584b16dd1c96a5b9455ac33eec057c560195e2daf18850d68c60f4df9ab9fd> (3904) - *A* appglue.site/src/Components/TopBarNav/TopBarNav.stories.tsx <https://github.com/x365Project/appglue/pull/128/files#diff-57395f420b75e2f1952218959da13f509280c2d589ff44cdeb169f079cfbe771> (33) - *M* appglue.site/src/Components/TopBarNav/TopBarNav.test.tsx <https://github.com/x365Project/appglue/pull/128/files#diff-c4349db6035394e0e8f024716e4b96c498d73274a76c29f00788f137b94da327> (242) - *M* appglue.site/src/Components/TopBarNav/__snapshots__/TopBarNav.test.tsx.snap <https://github.com/x365Project/appglue/pull/128/files#diff-d4616297803ab8ef01256c0246f2745fa6892e3b6490469b08d1ce4d50317894> (4828) - *A* appglue.site/src/Components/common/AppHeader/AppHeader.stories.tsx <https://github.com/x365Project/appglue/pull/128/files#diff-81183aac2e240bb2db21182ad06445bfbdcd83220108f2c5eb7dcba1e425fafd> (45) - *M* appglue.site/src/Components/common/AppHeader/AppHeader.test.tsx <https://github.com/x365Project/appglue/pull/128/files#diff-171672af6c4c79471669a6f4ac4372008daa7936f17f33b82e6af1a79f12d51d> (223) - *M* appglue.site/src/Components/common/AppHeader/__snapshots__/AppHeader.test.tsx.snap <https://github.com/x365Project/appglue/pull/128/files#diff-dd6007b0c64c586ff144908adfc4f3cf017772ef7411929d1f01d8c20916bdb6> (4756) - *A* appglue.site/src/Components/common/ChatOptions/ChatOptions.stories.tsx <https://github.com/x365Project/appglue/pull/128/files#diff-8f191b868b60a5d065fbf734ecdd0ebd6cfa85ed568d190f75203e897d526fdb> (17) - *A* appglue.site/src/Components/common/ChatOptions/ChatOptions.test.tsx <https://github.com/x365Project/appglue/pull/128/files#diff-0e2ef8e3d1be4bd994dfef22b4cf80475a1a3ac406ae3188dd99eb7327d18b39> (24) - *A* appglue.site/src/Components/common/ChatOptions/__snapshots__/ChatOptions.test.tsx.snap <https://github.com/x365Project/appglue/pull/128/files#diff-b8a56be555b7009aa244907412b9c03a61df92a616fe11273d457731acca80a7> (229) - *A* appglue.site/src/Components/common/MainContentArea/ContentArea.test.tsx <https://github.com/x365Project/appglue/pull/128/files#diff-03ceb229876408b0c264e535002fbc1f05f5a69bc655f974cf83f813dfcb2e22> (12) - *M* appglue.site/src/Components/common/MainContentArea/ContentArea.tsx <https://github.com/x365Project/appglue/pull/128/files#diff-6e17b681694b0253f30461b0a30968c880ffb0272a0218e57df9e9203e4c20fd> (2) - *A* appglue.site/src/Components/common/MainContentArea/MainContentArea.stories.tsx <https://github.com/x365Project/appglue/pull/128/files#diff-58552dd7d4fb4aed2359d51d18c79daabd798360ad895647bf73e454fc3639ae> (23) - *A* appglue.site/src/Components/common/MainContentArea/__snapshots__/ContentArea.test.tsx.snap <https://github.com/x365Project/appglue/pull/128/files#diff-f482df38435eb6a44ed8cefae9ca6bd355d2493a0523edf018b789003eae3d05> (15) - *A* appglue.site/src/Components/common/NotificationsOptions/NotificationsOptions.stories.tsx <https://github.com/x365Project/appglue/pull/128/files#diff-aed5357c5a961b21bd674849492163685ef78e71289a2ec11baa99056da7e402> (16) - *A* appglue.site/src/Components/common/NotificationsOptions/NotificationsOptions.test.tsx <https://github.com/x365Project/appglue/pull/128/files#diff-fe6fda07a58fe75d5b2a1a7632eba0e1054eb536717eded84a6808624822d1ac> (34) - *A* appglue.site/src/Components/common/NotificationsOptions/__snapshots__/NotificationsOptions.test.tsx.snap <https://github.com/x365Project/appglue/pull/128/files#diff-2c2fd4f00900e67fe985cfbc9f81aeb2f2f12423d38f0ca011246a3ab03ab2c4> (304) - *A* appglue.site/src/Components/common/Profile/ProfileComponent.stories.tsx <https://github.com/x365Project/appglue/pull/128/files#diff-63f5514754ee98c4501fe63fcd676157fa2f1dd568ee020dc4d7636c0efe2964> (16) - *A* appglue.site/src/Components/common/Profile/ProfileComponent.test.tsx <https://github.com/x365Project/appglue/pull/128/files#diff-b2020e2abe0aa2e3b7f0e7875169512c8a074b99eff38acc34a85901ff12037e> (22) - *A* appglue.site/src/Components/common/Profile/__snapshots__/ProfileComponent.test.tsx.snap <https://github.com/x365Project/appglue/pull/128/files#diff-9271f9a22569ed11e0bdea2647260b7222c9b845f0af7a27f35c7222d15212b4> (131) - *A* appglue.site/src/Components/common/SearchBar/SearchBar.stories.tsx <https://github.com/x365Project/appglue/pull/128/files#diff-58398dbcc554dc52a843d181031411c3867e0302982a990b7f9159e6bb41cbec> (13) - *A* appglue.site/src/Components/common/SettingsPanel/SettingsPanel.stories.tsx <https://github.com/x365Project/appglue/pull/128/files#diff-777bcd72bbcefae203b8fa67889ee8417e5fbde06c53640de77acb49dcc3c2ca> (25) - *A* appglue.site/src/Components/common/SingleSetting/SingleSetting.stories.tsx <https://github.com/x365Project/appglue/pull/128/files#diff-1656010f57225e2e86ea3ebf9b33849685d01d9ea7a466f5b992665fe9da8472> (19) Patch Links: - https://github.com/x365Project/appglue/pull/128.patch - https://github.com/x365Project/appglue/pull/128.diff — You are receiving this because you are subscribed to this thread. Reply to this email directly, view it on GitHub <#128>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/ARCCXOTXWAWUMOMBTHWRX5DTGMOINANCNFSM42EUHCWQ> .

Waleed9042 commented:
yeap, replied sir
